docs/topics/azure-stack.md

```diff
@@ -300,7 +300,9 @@ There is no official support for private-preview Kubernetes cluster with Windows
 
 ### Upgrading Kubernetes clusters created with the Ubuntu 16.04 distro
 
-Starting with AKS Engine v0.63.0, the Ubuntu 16.04 distro is not longer a supported option as the OS reached its end-of-life. In order to upgrade a cluster, make sure to set the OS distro to `aks-ubuntu-18.04` in your input API model, the one generated by `aks-engine deploy` and passed as input to `aks-engine upgrade`.
+Starting with AKS Engine v0.63.0, the Ubuntu 16.04 distro is not longer a supported option as the OS reached its end-of-life. For kubernetes clusters created with `aks-ubuntu-16.04` distro, it will automatically get upgrade to `aks-ubuntu-18.04`, corresponding distro values in `masterProfile` and `agentPoolProfile`s of the API model will be updated as well.
```

Comment:
I think that this text gives the idea that v0.63.0 will overwrite the distro value, which is not true.